Month1/Week2/Section1/Exercise1/main1.cpp

```diff
@@ -0,0 +1,42 @@
+/*Exercise 1 - Understanding classes
+Create a class called FirstClass with a public constructor and destructor. Have the constructor output �I�m in the constructor!� and have the destructor output �I�m in the destructor!�. Then, instantiate 5 classes and run your program, are the constructor and destructor calls where you expected them to be?
+
+Now create a loop of at least five iterations. For each iteration, instantiate your class once. Run your program, are the constructor and destructor calls where you expected them to be?
+
+*/
+#include <iostream>
+
+#include<string>
+
+class FirstClass
+{
+public:
+	std::string Texto;
```

Comment:
Remember all variables and comments should be in English. 

Please fix this for all exercises

Reply:
I change the word